strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmUser.java

```diff
@@ -0,0 +1,109 @@
+package org.carlspring.strongbox.controllers.layout.npm;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class NpmUser
+{
+
+    private String id;
+
+    private String name;
+
+    private String password;
+
+    private String type;
+
+    private String[] roles;
+
+    private String date;
+
+    private String email;
```

Comment:
@LeifBrooks do we really need all this additional fields? I mean that we use only `name` and `password`, right?

Reply:
No I think they can be removed.

Reply:
I think we should drop them in this case

Reply:
will do

Reply:
I removed the unused fields and methods